pkg/restapi/governance/operation/operations_test.go

```diff
@@ -427,7 +427,8 @@ func serveHTTPMux(t *testing.T, handler Handler, reqBytes []byte,
 	return rr
 }
 
-func serveHTTP(t *testing.T, handler http.HandlerFunc, method, path string, req []byte) *httptest.ResponseRecorder { // nolint: unparam,lll
+//nolint:unparam
```

Comment:
Can we use `_` for unused param and remove this lint ?

Reply:
there is no unused param here, it complains about "method" always being "POST"

Reply:
Oh ok.

Reply:
so either keep it or remove 'method' and hardcode "POST"

Reply:
I guess, we can keep it as its a test file.